src/api/goods.ts

```diff
@@ -6,6 +6,10 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return getAll().then(goods => goods.slice(0, 5));
+};
```

Comment:
You not only have to return first 5 but
Load 5 first goods shows 5 first goods after sorting them by name